scripts/autoblog.js

```diff
@@ -0,0 +1,145 @@
+let entryCopy2 = document.createElement('p');
+entryCopy2.textContent = 'Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui of cae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat. Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.';
+
+//ENTRY 3
+
+//create article w/ class 'post'
+let post3 = document.createElement('article');
+post3.setAttribute('class', 'post')
+
+//create image w/ src images/blog-3.jpg
+let img03 = document.createElement('img');
+img03.setAttribute('src', 'images/blog-3.jpg')
+img03.setAttribute('alt', 'a dog and its owner play fetch at sunset')
+img03.setAttribute('width', 275)
+img03.setAttribute('height', 260)
+
+//create div (blog text container)
+let blogCopyContainer3 = document.createElement('div');
+
+//create blog entry title
+let entryTitle3 = document.createElement('h4');
+entryTitle3.textContent = 'Teach Your Dog To Fetch!';
+
+//create blog body copy
+let entryCopy3 = document.createElement('p');
+entryCopy3.textContent = 'Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui of cae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat. Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.';
+//Blog Entries END
+
+//object of arrays. arrays contain blog posts
+const blogObjects = [
+{ post: document.createElement('article').setAttribute('class', 'post'),
+  image: img01,
+  copyContainer: blogCopyContainer1,
+  title: entryTitle1,
+  bodyCopy: entryCopy1,}
+];
+
+// Blog objects. 1 object per blog post.
+
+
+const blogObject2 = {
+  post: post2,
+  image: img02,
+  copyContainer: blogCopyContainer2,
+  title: entryTitle2,
+  bodyCopy: entryCopy2,
+};
+const blogObject3 = {
+  post: post3,
+  image: img03,
+  copyContainer: blogCopyContainer3,
+  title: entryTitle3,
+  bodyCopy: entryCopy3,
+};
+
+// Blog objects listed in an array (for loop)
+const objArray = [blogObject1, blogObject2, blogObject3];
+
```

Comment:
Since you aren't referencing your blog objects individually. You can actually just create them inside of the array:
```
const blogPosts = [
   {
      // post 1
   },
   {
      // post 2
   },
   {
      // post 3
   }
]
```

Reply:
Cool, that hadn't occurred to me, thanks! Would this create neater code or are there potential performance gains with this way?